docs/guides/external-mgs/readme.md

```diff
@@ -0,0 +1,151 @@
+---
+authors: Matt Richerson <matthew.richerson@hpe.com>
+categories: provisioning
+---
+
+# Lustre External MGS
+
+## Background
+
+Lustre has a limitation where only a single MGT can be mounted on a node at a single time. In the NNF cluster, this means the total number of Lustre file systems is limited to the number of NNF nodes. It also places extra scheduling constraints on the WLM to ensure that the MGTs don't conflict.
+
+A Lustre MGS is able to provide management for multiple separate Lustre file systems. The NNF software can be configured to take advantage of this to extend the maximum number of Lustre file systems beyond the number of NNF nodes. To do this, one or more Lustre MGSs must be specified that have a lifetime longer than any of the Lustre file systems that are using it. This can be done in a few different ways hrough the NNF software:
+
+1. Use a Lustre MGS from outside the NNF cluster
+2. Create a persistent Lustre file system through DWS and use the MGS it provides
+3. Create a pool of standalone persistent Lustre MGTs, and have the NNF software select one of them as the MGS
+
+These three methods are not mutually exclusive on the system as a whole. Individual file systems can use any of options 1-3 or create their own MGT.
+
+## Configuration with an External MGS
+
+An existing MGS external to the NNF cluster can be used to manage the Lustre file systems on the NNF nodes. An advantage to this configuration is that the MGS can be highly available. A disadvantage is that there is only a single MGS. An MGS serving more than a handful of Lustre file systems is not a common use case, so the Lustre code may prove less stable. Also, some Lustre operations from the clients may be slower if the MGS is located outside of the high speed network the compute nodes are on.
+
+The following yaml provides an example of what the `NnfStorageProfile` should contain to use an external MGS.
+
+```yaml
+apiVersion: nnf.cray.hpe.com/v1alpha1
+kind: NnfStorageProfile
+metadata:
+  name: external-mgs
+  namespace: nnf-system
+data:
+[...]
+  lustreStorage:
+    externalMgs: 1.2.3.4@eth0
+    combinedMgtMdt: false
+    standaloneMgtPoolName: ""
+[...]
+```
+
+## Configuration with Persistent Lustre
+
+The MGS from a persistent Lustre file system hosted on the NNF nodes can also be used as an external MGS. This configuration has the advantage of not relying on any hardware outside of the cluster as well as hosting the MGS inside the high speed network. However, there is no high availability, and a single MGS is still serving all the Lustre file systems created on the cluster.
+
+To configure a persistent Lustre file system that can share its MGS, a `NnfStorageProfile` should be used that does not specify an external MGS. The MGT can either share a volume with the MDT or not (`combinedMgtMdt`).
```

Comment:
If the MDT is shared on the MGT, then you can't use this MGS for multiple filesystems, can you?